app/views/admin/editions/_locale_fields.html.erb

```diff
@@ -1,39 +1,37 @@
-<% if edition.locale_can_be_changed? %>
-  <div class="edition-form--locale-fields">
-    <%= render "govuk_publishing_components/components/checkboxes", {
-      name: "edition[create_foreign_language_only]",
-      id: "edition_create_foreign_language_only",
-      heading: "Foreign language only #{edition.model_name.human.downcase}",
-      heading_size: "l",
-      no_hint_text: true,
-      error_items: errors_for_input(edition.errors, :create_foreign_language_only),
-      items: [
-        {
-          label: "Create a foreign language only #{edition.model_name.human.downcase}",
-          value: "1",
-          checked: form.object.primary_locale != "en",
-          conditional: (render "govuk_publishing_components/components/select", {
-            id: "edition_primary_locale",
-            name: "edition[primary_locale]",
-            label: "Document language",
-            heading_size: "s",
-            full_width: true,
-            allow_blank: true,
-            errors: errors_for(edition.errors, :primary_locale),
-            options: ([nil] + options_for_locales(Locale.non_english)).map do |language, value|
-              {
-                text: language,
-                value: value,
-                selected: edition.primary_locale == value
-              }
-            end
-          })
-        }
-      ]
-    } %>
+<div class="app-view-edit-edition__locale-field <%= "app-view-edit-edition__locale-field--hidden" unless edition.locale_can_be_changed? && edition.world_news_story? %>">
```

Comment:
sorry i missed this before. This field should render for Consultations and NewsArticles so this could be updated to 

```
edition.locale_can_be_changed? && (edition.is_a?(Consultation) || edition.world_news_story?)
```